ansible/roles/acs/tasks/discos_dependencies.yml

```diff
@@ -46,17 +46,18 @@
 
 - name: Check if the fitsviewer binary exist in the local repository
   stat:
-    path: "{{ local_repository_path }}/{{ fv_file }}"
+    path: "{{ local_repository_path }}/{{ fv.file }}"
   delegate_to: localhost
   run_once: true
```

Comment:
True?

Reply:
Both true and True are fine for Ansible, you can also use Yes or yes, sometimes I change the old `true` variables to `True` in order to be more consistent when I update parts of the code, but seems like this time I didn't update them. If we want to keep this kind of consistency too we should open a issue just for this.